src/mono/wasm/runtime/method-calls.ts

```diff
@@ -196,8 +191,6 @@ export function mono_bind_static_method(fqn: string, signature?: string/*ArgsMar
 }
 
 export function mono_bind_assembly_entry_point(assembly: string, signature?: string/*ArgsMarshalString*/): Function {
-    bindings_lazy_init();// TODO remove this once Blazor does better startup
```

Comment:
we may want to, as a transitional step, have bindings_lazy_init still be here but throw instead of doing the init. that way any old code doing the wrong thing will fail loudly and fast